pkgs/misc/vim-plugins/update.py

```diff
@@ -477,13 +482,27 @@ def parse_args():
         default=DEFAULT_OUT,
         help="Filename to save generated nix code",
     )
+    parser.add_argument(
+        "--commit",
+        dest="commit",
+        action="store_true",
+        help="Automatically commit updates",
+    )
 
     return parser.parse_args()
 
 
-def main() -> None:
+def get_nixpkgs_repo() -> git.Repo:
+    repo = git.Repo(NIXPKGS_PATH)
```

Comment:
We can get rid of `NIXPKGS_PATH` by using `git.Repo(".", search_parent_directories=True)`.

Reply:
I would think if we used `ROOT` rather than `.` we could avoid the assumption that the current working directory is `vim-plugins`.

Reply:
Good idea, that is better.